src/components/RadioButtonGroup/styled/buttonItem.js

```diff
@@ -1,61 +1,67 @@
 /* stylelint-disable max-line-length */
 import styled from 'styled-components';
 import { BORDER_RADIUS_2 } from '../../../styles/borderRadius';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
 
-const StyledButtonItem = styled.span`
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-    height: 2.5rem;
-    border-radius: ${BORDER_RADIUS_2};
-    border: solid 1px transparent;
-    transition: transform 600ms cubic-bezier(0.02, 0.94, 0.09, 0.97),
-        all 300ms cubic-bezier(0.17, 0.67, 0.14, 1.03);
-    transform: translate3d(0, 0, 0);
+const StyledButtonItem = attachThemeAttrs(styled.span)`
+    ${props => `
+        display: inline-flex;
+        justify-content: center;
+        align-items: center;
+        height: 2.5rem;
+        border-radius: ${BORDER_RADIUS_2};
+        border: solid 1px transparent;
+        transition: transform 600ms cubic-bezier(0.02, 0.94, 0.09, 0.97),
+            all 300ms cubic-bezier(0.17, 0.67, 0.14, 1.03);
+        transform: translate3d(0, 0, 0);
```

Comment:
there is already an open PR for @wildergd passing theme values to this component, in order to avoid complex conflicts I think you should only change the part related to inverse variant